tracing/src/lib.rs

```diff
@@ -174,6 +174,15 @@
 //! # fn main() {}
 //! ```
 //!
+//! For external functions which don't have built-in tracing support and can't have
```

Comment:
I'm on the fence about whether or not I would explicitly mention _external_ functions here — another potential use of `in_scope` is calling an internal function either with a span which can only be created in the calling scope (e.g. if it wants to record variables from the calling scope as fields) or which should be instrumented by a different span depending on which scope it's called from. I don't necessarily want to suggest that this is used _only_ for calling functions from other libraries.

Not a big deal, though.